terraform/devnet-11/nodes.tf

```diff
@@ -0,0 +1,296 @@
+  default = {
+    name            = "mev-relay"
+    count           = 1
+    validator_start = 0
+    validator_end   = 0
+  }
+}
+
+# Bootnode
+variable "bootnode" {
+  default = {
+    name            = "bootnode"
+    count           = 1
+    validator_start = 0
+    validator_end   = 0
+    size            = "s-4vcpu-16gb-amd"
+  }
+}
+
+# Lighthouse
+variable "lighthouse_geth" {
+  default = {
+    name            = "lighthouse-geth"
+    count           = 1
+    validator_start = 0
+    validator_end   = 100
+  }
+}
+
+variable "lighthouse_nethermind" {
+  default = {
+    name            = "lighthouse-nethermind"
+    count           = 1
+    validator_start = 100
+    validator_end   = 200
+  }
+}
+
+variable "lighthouse_besu" {
+  default = {
+    name            = "lighthouse-besu"
+    count           = 1
+    validator_start = 200
+    validator_end   = 225
+  }
+}
+
+variable "lighthouse_erigon" {
+  default = {
+    name            = "lighthouse-erigon"
+    count           = 1
+    validator_start = 225
+    validator_end   = 250
+  }
+}
+
+variable "lighthouse_ethereumjs" {
+  default = {
+    name            = "lighthouse-ethereumjs"
+    count           = 1
+    validator_start = 250
+    validator_end   = 260
+  }
+}
+
+# Prysm
+variable "prysm_geth" {
+  default = {
+    name            = "prysm-geth"
+    count           = 1
+    validator_start = 260
+    validator_end   = 360
+  }
+}
+
+variable "prysm_nethermind" {
+  default = {
+    name            = "prysm-nethermind"
+    count           = 1
+    validator_start = 360
+    validator_end   = 460
+  }
+}
+
+variable "prysm_besu" {
+  default = {
+    name            = "prysm-besu"
+    count           = 1
+    validator_start = 460
+    validator_end   = 485
+  }
+}
+
+variable "prysm_erigon" {
+  default = {
+    name            = "prysm-erigon"
+    count           = 1
+    validator_start = 485
+    validator_end   = 510
+  }
+}
+
+variable "prysm_ethereumjs" {
+  default = {
+    name            = "prysm-ethereumjs"
+    count           = 1
+    validator_start = 510
+    validator_end   = 520
+  }
+}
+
+# Lodestar
+variable "lodestar_geth" {
+  default = {
+    name            = "lodestar-geth"
+    count           = 1
+    validator_start = 520
+    validator_end   = 620
+  }
+}
+
+variable "lodestar_nethermind" {
+  default = {
+    name            = "lodestar-nethermind"
+    count           = 1
+    validator_start = 620
+    validator_end   = 720
+  }
+}
+
+variable "lodestar_besu" {
+  default = {
+    name            = "lodestar-besu"
+    count           = 1
+    validator_start = 720
+    validator_end   = 745
+  }
+}
+
+variable "lodestar_erigon" {
+  default = {
+    name            = "lodestar-erigon"
+    count           = 1
+    validator_start = 745
+    validator_end   = 770
+  }
+}
+
+variable "lodestar_ethereumjs" {
+  default = {
+    name            = "lodestar-ethereumjs"
+    count           = 1
+    validator_start = 770
```

Comment:
do we want so many ethjs nodes if they cant handle it? we may as well just have one and call it?

Reply:
ok